src/all/taddyink/src/eu/kanade/tachiyomi/extension/all/taddyink/TaddyInk.kt

```diff
@@ -0,0 +1,188 @@
+package eu.kanade.tachiyomi.extension.all.taddyink
+
+import androidx.preference.PreferenceScreen
+import androidx.preference.SwitchPreferenceCompat
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.network.interceptor.rateLimit
+import eu.kanade.tachiyomi.source.ConfigurableSource
+import eu.kanade.tachiyomi.source.model.Filter
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.HttpSource
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.Response
+import uy.kohesive.injekt.injectLazy
+
+open class TaddyInk(
+    override val lang: String,
+    private val taddyLang: String,
+) : ConfigurableSource, HttpSource() {
+
+    final override val baseUrl = "https://taddy.org"
+    override val name = "Taddy INK (Webtoons)"
```

Comment:
Same here if accepted.

```suggestion
    override val name = "Taddy"
```

Reply:
I would prefer to keep Taddy INK (Webtoons) vs Taddy.